packages/esm-patient-chart-app/src/visit/start-visit-button.component.tsx

```diff
@@ -6,17 +6,23 @@ import { showSnackbar } from '@openmrs/esm-framework';
 
 interface StartVisitButtonProps {
   patientUuid: string;
+  handleBackToSearchList?: () => void;
```

Comment:
Can we assign a more generic name for this? Something like `handleCloseVisitWorkspace`